python-models/server/server.py

```
@@ -0,0 +1,223 @@
+
+    _MODELS = registry
+    return registry
+
+
+@app.get("/health")
+def health() -> Dict[str, str]:
+    try:
+        _ = _load_models()
+        return {"status": "ok"}
+    except Exception as e:  # noqa: BLE001
+        raise HTTPException(status_code=500, detail=str(e))
```

Ruff (0.12.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)